spec/acceptance/mod_passenger_spec.rb

```diff
@@ -137,10 +150,7 @@ class { 'apache::mod::passenger': }
         case fact('operatingsystem')
         when 'Ubuntu'
           case fact('lsbdistrelease')
-          when '14.04'
-            it { is_expected.to contain %(PassengerDefaultRuby "#{passenger_default_ruby}") }
-            it { is_expected.not_to contain '/PassengerRuby/' }
-          when '16.04'
+          when '14.04', '16.04', '18.04'
             it { is_expected.to contain %(PassengerDefaultRuby "#{passenger_default_ruby}") }
             it { is_expected.not_to contain '/PassengerRuby/' }
           else
```

Comment:
The else block is unsupported as well, same as earlier.

Reply:
fixed